index.html

```diff
@@ -0,0 +1,293 @@
+                           <a href="#"><img src="images/mIcon.svg" alt=""></a>
+                       </li>
+                       <li>
+                           <a href="#"><img src="images/gitHubIcon.svg" alt=""></a>
+                       </li>
+                       <li>
+                           <a href="#"><img src="images/peaceIcon.svg" alt=""></a>
+                       </li>
+                   </ul>
+               </div>
+               <br>
+               <button type="button"><img src="images/Getmyresume.png" alt=""></button>
+               </div>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Languages.png" alt=""></li>
+                   <li class="greaterthan"><a href="">&#86;</a></li>
+                </ul>
+               </div>
+               <div class="frameworks">
+                   <ul>
+                       <li><img src="images/Frame9.png" alt=""></li>
+                       <li><img src="images/Frame10.png" alt=""></li>
+                       <li><img src="images/Frame11.png" alt=""></li>
+                   </ul>
+               </div>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Frameworks1.png" alt=""></li>
+                   <li class="lang-text"><a href="">&#62;</a></li>
+                </ul>
+               </div>
+               <p><img src="images/SeparatorBottom.png" alt=""></p>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Skills1.png" alt=""></li>
+                   <li class="skills-text lang-text"><a href="">&#62;</a></li>
+                </ul>
+               </div>
+               <p><img src="images/SeparatorBottom.png" alt=""></p>
+           </div>       
+           </section>
+           <section class="works-2">
+            <div class="headline1">
+              
+                 <h2 class="mainheader">
+                     Contact Me</h2>
+                 <p class="details-2">
+                   <img src="images/PrimaryText3.png" alt="">
+                 </p>
+               
+               <div class="form-input">
+                <form action="https://formspree.io/f/mayvkezq" method="post">
+                    <div class="container">
+                      <input type="text" placeholder="Name" name="Name" id="name" maxlength="30">                
+
+                      <input type="email" placeholder="Email" name="email" id="email" required>
+                  
+                      <textarea class="textbox" placeholder="Comments" name="comments" id="comments" maxlength="500"></textarea>                                      
+                      
+                    </div>           
+                    <button type="submit" class="registerbtn"><img src="images/GetItTouch.png" alt=""></button>    
+                    
+                  </form>
```

Comment:
- [ ] Form works well, it meets the requirements of validations and input types. But the design is different from the one provided in Figma: 

Your design  | Figma design
------------- | -------------
<img width="408" alt="Screenshot 2022-06-08 at 18 35 48" src="https://user-images.githubusercontent.com/77038610/172643879-fd42f6da-eb00-478b-bbf2-10c98da2a66d.png">  | <img width="310" alt="Screenshot 2022-06-08 at 18 35 29" src="https://user-images.githubusercontent.com/77038610/172643807-8d4f0a88-5653-4d20-923d-82318bbf6df2.png">

### Please note that for this project you have to strictly stick to Figma template design, 

- [ ] Please make sure that the form stretches from theft to right without whitespace around.
- [ ] Kindly follow the design guidelines, set the same colors, font size, margins paddings. 
- [ ] Please set less border radius on input elements,